### PR TITLE
update auxo

### PR DESCRIPTION
**Pull request by AmberLJC**
#161

### CODE REVIEW

File: source/_data/SymbioticLab.bib [hunk 1]
```diff
@@ -1260,4 +1279,4 @@ @InProceedings{modelkeeper:nsdi23
     ModelKeeper achieves 1.3x-4.3x faster training completion
     with little overhead and no reduction in model accuracy.
   }
-}
+}
```
Comment (jaywonchung) [inline source/_data/SymbioticLab.bib:1282]:
```suggestion
}
```
Lines should end with a newline character.

File: source/_data/SymbioticLab.bib [hunk 2]
```diff
@@ -1095,6 +1095,25 @@ @article{swan:arxiv22
...
+	  title={Auxo: Heterogeneity-Mitigating Federated Learning via Scalable Client Clustering},
+	  author={Jiachen Liu, Fan Lai, Yinwei Dai, Aditya Akella, Harsha Madhyastha, Mosharaf Chowdhury},
+	  archiveprefix = {arXiv},
+	  eprint= {},
+	  url= {https://arxiv.org/abs/2210.16656},
+	  year={2022},
+	  month={Oct},
+	  publist_confkey = {arXiv:2210.16656},
+	  publist_link = {paper || auxo-arxiv22.pdf},
+	  publist_topic = {Systems + AI},
```
Comment (jaywonchung) [inline source/_data/SymbioticLab.bib:1109]: Shouldn't it have Wide-Area Computing?

**Review by jaywonchung — CHANGES_REQUESTED** [review]
Did some nitpicking.

File: source/_data/SymbioticLab.bib (hunk 2 above)
Comment (AmberLJC) [reply source/_data/SymbioticLab.bib:1109]: Yep, thanks

File: source/_data/SymbioticLab.bib (hunk 1 above)
Comment (AmberLJC) [reply source/_data/SymbioticLab.bib]: i cannot edit it to be this on my side?

File: source/_data/SymbioticLab.bib [hunk 3]
```diff
@@ -1150,7 +1170,7 @@ @Article{zeus:arxiv22
 
     In this paper, we observe that common practices to improve training performance can often lead to inefficient energy usage. More importantly, we demonstrate that there is a tradeoff between energy consumption and performance optimization. To this end, we propose an optimization framework, Zeus, to navigate this tradeoff by automatically finding optimal job- and GPU-level configurations for recurring DNN training jobs. Zeus uses an online exploration-exploitation approach in conjunction with just-in-time energy profiling, averting the need for expensive offline measurements, while adapting to data drifts over time. Our evaluation shows that Zeus can improve the energy efficiency of DNN training by 15.3%--75.8% for diverse workloads.
   }
-}
+}
```
Comment (jaywonchung) [inline source/_data/SymbioticLab.bib:1173]:
```suggestion
}
```

**Review by jaywonchung — APPROVED** [review]
Removed one trailing whitespace. LGTM.

**Pull request merged**